.gitignore

```diff
@@ -134,3 +134,5 @@ tests/dist/
 .repository/
 apps/console/src/extensions/i18n/tmp/**
 .nx
+apps/myaccount/v8-compile-cache-0/**
```

Comment:
remove this and not add the files to the repo as well. these are temp files

Reply:
addressed with [cb35632](https://github.com/wso2/identity-apps/pull/6027/commits/cb35632c79a886962c53ebf5a56d90a028f6e909)